src/yuv_rgb/transfer.rs

```diff
@@ -331,7 +331,7 @@ fn arib_b67_inverse_oetf(x: f32) -> f32 {
     if x <= 0.5 {
         (x * x) * (1.0 / 3.0)
     } else {
-        (((x - ARIB_B67_C) / ARIB_B67_A).exp() + ARIB_B67_B) / 12.0
+        expf(((x - ARIB_B67_C) / ARIB_B67_A) + ARIB_B67_B) / 12.0
```

Comment:
I think the parens are in the wrong place here, the `+ ARIB_B67_B` should not be included in the `expf` call

Reply:
Huh, you're right. How did the tests not catch this..? I'll take another look tonight as to how this didn't fail the tests

Reply:
Are there tests around HLG? That might be the answer... :sweat_smile:

Reply:
[There are tests](https://github.com/rust-av/yuvxyb/blob/7c3edaf8bb2361b1aea73c6f5c3fe1906261150b/src/lib.rs#L206-L237), I did check it beforehand. What I did not expect was the tests [not actually verifying the result](https://github.com/rust-av/yuvxyb/blob/7c3edaf8bb2361b1aea73c6f5c3fe1906261150b/src/lib.rs#L295) of the conversions.

Might be a good idea to implement at some point..

Reply:
Interesting... not really sure how that happened. Whoops.